src/test/java/com/wikia/webdriver/common/core/CommonExpectedConditions.java

```diff
@@ -130,7 +124,6 @@ public Boolean apply(WebDriver driver) {
    * it.
    *
    * @param givenElement element to be checked
-   * @author Michal Nowierski
    */
   public static ExpectedCondition<WebElement> elementNotToBeClickable(final WebElement givenElement) {
     return new ExpectedCondition<WebElement>() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this anonymous class number of lines from 23 to at most 20, or make it a named class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1188)